tools/minicluster/minicluster.go

```diff
@@ -34,7 +36,7 @@ func main() {
 	// aps indicates how many actions to be injected in one second. Default is 0
 	var aps int
 
-	flag.IntVar(&timeout, "timeout", 300, "duration of running nightly build")
+	flag.IntVar(&timeout, "timeout", 100, "duration of running nightly build")
```

Comment:
Why changing it to 100?

Reply:
Sorry, it is just used for quickly test, I forgot to undo it before submit.